sdk/vision/azure-ai-vision-imageanalysis/src/customizations/customizations.md

```diff
@@ -138,12 +134,12 @@ import java.net.URL;
      * <!-- end com.azure.ai.vision.imageanalysis.async-analysis-from-url -->
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<ImageAnalysisResult> analyze(URL imageUrl, List<VisualFeatures> visualFeatures,
+    public Mono<ImageAnalysisResult> analyzeFromUrl(String imageUrl, List<VisualFeatures> visualFeatures,
```

Comment:
I noticed in the JavaDoc that we specify "At least one visual feature must be specified" but we don't check if `visualFeatures` list is null or empty. Is that something the service will validate and throw a 400 error?

Reply:
@srnagar yes, good question, this is something I verified a while back. I made sure this is still true just now :-) If you provide an empty visual features list, the service returns a proper error, which gets translated to this exception:
```
Exception in thread "main" com.azure.core.exception.HttpResponseException: Status code 400, "{"error":{"code":"InvalidRequest","message":"Either 'features' or 'model-name' needs to be specified in the query parameter."}}"
```

The error code is somewhat out of date however, because the "or 'model-name'" part of the error message no longer applies to the GA version of the REST API (this has to do with the application providing a custom AI model name, which is a feature removed from the GA REST API). I will inform the service folks about this issue.